pkg/cli/secretsencrypt/secrets_encrypt.go

```diff
@@ -0,0 +1,197 @@
+	if nodeName == "" {
+		nodeName, err = os.Hostname()
+		if err != nil {
+			return controlConfig, err
+		}
+	}
+
+	os.Setenv("NODE_NAME", nodeName)
+
+	controlConfig.DataDir, err = server.ResolveDataDir(cfg.DataDir)
+	if err != nil {
+		return controlConfig, err
+	}
+	if cmds.ServerConfig.ServerURL == "" {
+		cmds.ServerConfig.ServerURL = "https://127.0.0.1:6443"
+	}
+
+	if cmds.ServerConfig.Token == "" {
+		fp := filepath.Join(controlConfig.DataDir, "token")
+		tokenByte, err := ioutil.ReadFile(fp)
+		if err != nil {
+			return controlConfig, err
+		}
+		controlConfig.Token = string(bytes.TrimRight(tokenByte, "\n"))
+	} else {
+		controlConfig.Token = cmds.ServerConfig.Token
+	}
+	controlConfig.EncryptForce = cfg.EncryptForce
+
+	return controlConfig, nil
+}
+
+func Run(app *cli.Context) error {
+	fmt.Println("This command does nothing, use the subcommands")
+	return nil
+}
+
+func Enable(app *cli.Context) error {
+	var err error
+	if err = cmds.InitLogging(); err != nil {
+		return err
+	}
+	controlConfig, err := commandPrep(app, &cmds.ServerConfig)
+	if err != nil {
+		return err
+	}
+	info, err := clientaccess.ParseAndValidateTokenForUser(cmds.ServerConfig.ServerURL, controlConfig.Token, "node")
+	if err != nil {
+		return err
+	}
+	if err = info.Put("/v1-" + version.Program + "/encrypt-enable"); err != nil {
+		return err
+	}
+	fmt.Println("secrets-encryption enabled")
+	return nil
+}
+
+func Disable(app *cli.Context) error {
+	var err error
+	if err = cmds.InitLogging(); err != nil {
+		return err
+	}
+	controlConfig, err := commandPrep(app, &cmds.ServerConfig)
+	if err != nil {
+		return err
+	}
+	info, err := clientaccess.ParseAndValidateTokenForUser(cmds.ServerConfig.ServerURL, controlConfig.Token, "node")
+	if err != nil {
+		return err
+	}
+	if err = info.Put("/v1-" + version.Program + "/encrypt-enable"); err != nil {
+		return err
+	}
+	fmt.Println("secrets-encryption disabled")
+	return nil
+}
+
+func Status(app *cli.Context) error {
+	if err := cmds.InitLogging(); err != nil {
+		return err
+	}
+	controlConfig, err := commandPrep(app, &cmds.ServerConfig)
+	if err != nil {
+		return err
+	}
+	info, err := clientaccess.ParseAndValidateTokenForUser(cmds.ServerConfig.ServerURL, controlConfig.Token, "node")
+	if err != nil {
+		return err
+	}
+	data, err := info.Get("/v1-" + version.Program + "/encrypt-status")
+	if err != nil {
+		return err
+	}
+	fmt.Print(string(data))
+	return nil
+}
+
+func Prepare(app *cli.Context) error {
+	var err error
+	if err = cmds.InitLogging(); err != nil {
+		return err
+	}
+	controlConfig, err := commandPrep(app, &cmds.ServerConfig)
+	if err != nil {
+		return err
+	}
+	info, err := clientaccess.ParseAndValidateTokenForUser(cmds.ServerConfig.ServerURL, controlConfig.Token, "node")
+	if err != nil {
+		return err
+	}
+	if controlConfig.EncryptForce {
+		err = info.Put("/v1-" + version.Program + "/encrypt-prepare-force")
+	} else {
+		err = info.Put("/v1-" + version.Program + "/encrypt-prepare")
+	}
```

Comment:
Same as above for enabling/disabling, make this one endpoint with a EncryptionRotationSettings with Force and Step fields on it. Could possibly even be merged with the Status field and struct?
```suggestion
	if controlConfig.EncryptForce {
		err = info.Put("/v1-" + version.Program + "/encrypt/rotate")
	} else {
		err = info.Put("/v1-" + version.Program + "/encrypt/rotate")
	}
```